content/docs/user-guide/action-manager/fundamentals/concepts/actions.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Actions
+linkTitle: Actions
+description: 
+weight: 101
+---
+
+The Action Manager system provides interfaces to register actions and related frameworks to interact with the O3DE Editor.
+
+
+## Action Context
+
+An Action Context is an object that owns a collection of Actions.
+
+An Action Context is comprised of:
+
+* A string **identifier** that makes it possible to reference it in API calls;
+* A **name** string, which is meant to be a human-readable description to show in UI (currently unused);
```

Comment:
@AMZN-daimini  Is the name parameter unattached? I don't think we need to mention that here, if it's intended that name works
```suggestion
* A **name** string, which is meant to be a human-readable description to show in UI;
```